.spelling

```diff
@@ -162,7 +162,9 @@ DellOS
 DHCP
 dialog
 diskless
+DNs
```

Comment:
As you point out, this is more likely to be a typo of DNS than it is to be Distinguished Names. Because of that, I think we should either not use the acronym in our pages (and thus not need to add it to this file), or we should only put this as a file-specific exception at the end of the file.